common/reviews/api/heft.api.md

```diff
@@ -13,11 +13,11 @@ import { CommandLineFlagParameter } from '@rushstack/ts-command-line';
 import { CommandLineIntegerParameter } from '@rushstack/ts-command-line';
 import { CommandLineStringParameter } from '@rushstack/ts-command-line';
 import { IPackageJson } from '@rushstack/node-core-library';
-import { ITerminalProvider } from '@rushstack/node-core-library';
+import { ITerminal } from '@rushstack/terminal';
```

Comment:
We should combine these two @rushstack/terminal import statements.

Reply:
This is generated by API Extractor, so we don't really have control over this.